kong/plugins/rate-limiting/handler.lua

```diff
@@ -72,7 +80,7 @@ function RateLimitingHandler:access(conf)
 
   -- Consumer is identified by ip address or authenticated_credential id
   local identifier = get_identifier(conf)
-  local api_id = ngx.ctx.api.id
+  local api_id = get_api_id(conf)
```

Comment:
Could we avoid the function overhead here by just assigning this as

`local api_id =  conf.cross_api and "000..." or ngx.ctx.api.id`

?